task_sdk/src/airflow/sdk/definitions/taskgroup.py

```diff
@@ -622,6 +634,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         super().__exit__(exc_type, exc_val, exc_tb)
 
 
+# TODO: What do to about that ? (Should it be updated accordingly)?
```

Comment:
@ashb @kaxil 
This version of the `task_group_to_dict` will be removed in airflow 3 from core, and replaced with the new version that you can see in this PR. Should we also update this `task_group_to_dict` to the new format or does the `task_sdk` need this particular version to remain ?

Reply:
New version is fine, as long as test passes